frontend/src/app/facilitySelect/WithFacility.tsx

```diff
@@ -1,47 +1,63 @@
-import React from "react";
-import { useDispatch, useSelector, connect } from "react-redux";
+/* eslint-disable graphql/template-strings */
+import { useCallback, useMemo, FC, useEffect } from "react";
 import { useHistory } from "react-router-dom";
 
-import { RootState, updateFacility } from "../store";
+import { useFacilities } from "../../hooks/useFacilities";
+import { useAppConfig } from "../../hooks/useAppConfig";
 import { getFacilityIdFromUrl } from "../utils/url";
 
 import FacilityPopup from "./FacilityPopup";
 import FacilitySelect from "./FacilitySelect";
 
-const Loading: React.FC<{}> = () => <p>Loading facility information...</p>;
+const Loading: FC<{}> = () => <p>Loading facility information...</p>;
 
 interface Props {}
 
-const WithFacility: React.FC<Props> = ({ children }) => {
-  const dispatch = useDispatch();
+const WithFacility: FC<Props> = ({ children }) => {
```

Comment:
This component no longer works. 

Steps to reproduce
1. Add a second facility via http://localhost:3000/settings/add-facility/
2. go to http://localhost:3000/
3. click on a facility
4. you will still see the facility select screen

Expected behavior
the facility is selected and you are routed to the app

Reply:
With this I need some help, I can't add another facility. 
![Screenshot 2021-06-08 at 7 39 48 PM](https://user-images.githubusercontent.com/7571965/121224495-4a761d80-c891-11eb-876a-5e0a4e2d746d.png)